tests/lib/rules/require-expect.js

```diff
@@ -275,6 +275,16 @@ ruleTester.run("require-expect", rule, {
             options: ["except-simple"]
         },
 
+        // Multiple assert statements only report one error per test
+        {
+            code: "test('name', function(assert) { maybe(function() { assert.ok(true); assert.ok(true); }) });",
+            errors: [{
+                message: exceptSimpleErrorMessage("assert.expect"),
+                type: "CallExpression"
+            }],
+            options: ["except-simple"]
+        },
```

Comment:
Can we check line and column here as well, so we know that the `test()` CallExpression is the one reported on? (Lots of CallExpression nodes in this code)